packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -54,4 +57,13 @@ export class MeterProvider implements api.MeterProvider {
 
     return this._meters.get(key)!;
   }
+
+  shutdown(): void {
+    if (this._config['exporter']) {
+      this._config['exporter'].shutdown();
```

Comment:
I believe exporter shutdown takes a callback param? how will we know when its done?

Reply:
Is this another case where a user may want to call shutdown manually?

Reply:
yes. I can't think of a reason not to support the use-case

Reply:
Metric exporter does not take a callback param currently.

Reply:
hmm bummer